kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1540,6 +1525,27 @@ object KyuubiConf {
       .stringConf
       .createOptional
 
+  val SESSION_RESERVE_ON_BROKEN_ENABLED: ConfigEntry[Boolean] =
+    buildConf("kyuubi.session.reserve.on.broken.enabled")
+      .internal
+      .doc("Whether to reserve the session on the transport protocol broken. Now we support" +
+        " RetryingKyuubiSyncThriftClient, it will close the recreate the protocol for thrift" +
+        " request retry, and we need reserve the session in engine side for that.")
+      .version("1.6.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val SESSION_RESERVE_ON_BROKEN_TIMEOUT: ConfigEntry[Long] =
```

Comment:
`kyuubi.session.reserve.*` are unnecessary as a retry logic

Reply:
it is used to tell the engine front end service that do not close the session on protocol broken.

Otherwise, the session will be closed.

Reply:
the session will be closed on disconnected with these..

Reply:
yeah, so we need this config to reserve the old context.